Latest values

```diff
@@ -0,0 +1,73 @@
+
+1k:
+Scheduled 30000 Pods in 76 seconds (394 per second on average). min QPS was 0
+--- FAIL: TestSchedule100Node3KPods (92.18s)
+	scheduler_test.go:49: Failing: Scheduling rate was too low for an interval, we saw rate of 0, which is the allowed minimum of 30 ! 
+FAIL
+exit status 1
+FAIL	k8s.io/kubernetes/test/integration/scheduler_perf	92.315s(122.996)
```

Comment:
@ataturk. The number 92.315 is the scheduling duration. The one in brackets(122.996) is old value. Its same for every entry.